wremnants/datasets/datagroups.py

```diff
@@ -123,12 +123,12 @@ def setHists(self, baseName, syst, procsToRead=None, label=None, nominalIfMissin
                     logger.debug(f"Forcing group member {member.name} to read the nominal hist for syst {syst}")
 
                 try:
-                    h = self.readHist(baseName, member, read_syst, scaleOp=scale, forceNonzero=forceNonzero, scaleToNewLumi=scaleToNewLumi)
+                    h = self.readHist(baseName, member, procName, read_syst, scaleOp=scale, forceNonzero=forceNonzero, scaleToNewLumi=scaleToNewLumi)
```

Comment:
How is this used?

Reply:
For the fakes estimate, the signal procs have the extra gen bin which should be integrated over before subtracting it from the data in the anti-iso region. This is now done in the readHist function (datagroupsLowPU), but we need to pass the group name (= proc name here).